ios/DataManager.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DataManager.swift
+//  FlipDatePicker
+//
+//  Created by Halina Smolskaya on 10/06/2024.
+//
+
+import Foundation
+
+public struct DataManager {
+    let collections: [DataSource]
```

Comment:
shouldn't it be private?

Reply:
It cannot be `private` because we need to iterate through collections in the DatePicker. It's a read-only property so I think we're on the safe side here